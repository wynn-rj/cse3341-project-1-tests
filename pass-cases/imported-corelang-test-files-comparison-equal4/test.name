comparison equal4 (Imported from corelan-test-files)
