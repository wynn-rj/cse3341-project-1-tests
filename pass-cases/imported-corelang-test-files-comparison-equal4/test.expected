program
  int x;
begin
  if 984=948 then
    output 1;
  endif;
end
